phpBB/phpbb/textformatter/s9e/parser.php

```diff
@@ -237,6 +239,25 @@ public function get_errors()
 			$errors = array_map('unserialize', array_unique(array_map('serialize', $errors)));
 		}
 
+		$parser = $this;
+
+		/**
+		* Modify error messages generated by the s9e\TextFormatter's logger
+		*
+		* @event core.text_formatter_s9e_get_errors
+		* @var parser	parser		This parser service
+		* @var array	entries		s9e\TextFormatter's logger entries
+		* @var array	errors		Error arrays with language key and optional arguments
+		* @since 3.2.10-RC1
```

Comment:
Can we have a new line , between `@var` & `@since` , if possible...

Best regards 👍

Reply:
No other event has a newline there and I don't see a reason to add one.

Reply:
ok 👍